src/database/config/config.js

```diff
@@ -0,0 +1,13 @@
+require('dotenv').config();
+
+module.exports.development = {
+  url: process.env.DATABASE_URL,
+  dialect: 'postgres',
+  logging: false,
+};
+
+module.exports.test = {
+  url: process.env.TEST_DATABASE_URL,
+  dialect: 'postgres',
+  logging: false,
+};
```

Comment:
add a new line at the end of the file. And also you have to add configuration for `production`